etc/nginx.conf

```diff
@@ -33,7 +33,7 @@ http {
       add_header X-Permitted-Cross-Domain-Policies "";
       add_header X-Frame-Options "deny";
       add_header Referrer-Policy "no-referrer-when-downgrade";
-      add_header Content-Security-Policy "";
+      add_header Content-Security-Policy "connect-src 'self' *.amplitude.com *.zesty.io *.a.run.app *.tiny.cloud *.getbynder.com *.bynder.com d8ejoa1fys2rk.cloudfront.net *.sentry.io www.googleapis.com us-central1-zesty-dev.cloudfunctions.net;";
```

Comment:
What is the AWS d8ejoa1fys2rk.cloudfront.net URL? DM if this is customer information.

Reply:
Oh I forgot to add some comments to outline the non-obvious domains, but that one's for the bynder modules. I decided to not use a wildcard to make sure we're only allowing only that specifically